### PR TITLE
feat(ci): workflow for automatically assigning issues to GitHub project

### PR DESCRIPTION
**Pull request by vdovhanych**
This will automatically assign newly created issues to our `Backlog` project. We also have an option to set all pull requests to the designed project.

### CODE REVIEW

File: .github/workflows/project-automation.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: add-new-issues-to-repository-based-project-column
+        uses: docker://takanabe/github-actions-automate-projects:v0.0.1
+        if: github.event_name == 'issues' && github.event.action == 'opened'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_PROJECT_URL: https://github.com/trezor/trezor-suite/projects/1
+          GITHUB_PROJECT_COLUMN_NAME: 📥 Inbox
+    # We can activate this when we want to also add pull requests to some project in our repository.
```
Comment (tsusanka) [inline .github/workflows/project-automation.yml]: @matejkriz would like this for the [Suite PR project](https://github.com/orgs/trezor/projects/53)?

Btw let's move it to Suite repo as well from the organization. Also would like to consolidate it with [FW](https://github.com/trezor/trezor-firmware/projects/2)?

**Review by tsusanka — COMMENTED** [review]
LGTM but let's wait for @matejkriz.

File: .github/workflows/project-automation.yml (hunk 1 above)
Comment (matejkriz) [reply .github/workflows/project-automation.yml]: the issue with projects in firmware/suite repos is that we don't have everything inside those two monorepos yet :-/ How to put some issue from https://github.com/trezor/connect to suite backlog?

Comment (tsusanka) [reply .github/workflows/project-automation.yml]: Discussed with @matejkriz:
- We have killed the Suite PR project, it is not being used. @vdovhanych please remove the commented code then.
- Although it is unfrutunate that we can't add Connect issues now to the project we are fine with for now.

tl;dr: please remove the commentd code and let's merge.

Comment (vdovhanych) [reply .github/workflows/project-automation.yml]: done

**Review by tsusanka — APPROVED** [review]

**Comment by tsusanka** [comment]
After this is confirmed to work please make another one for Firmware.
